### PR TITLE
Forward redis metrics to the sumologic

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Forward redis metrics to the sumologic

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -1432,6 +1432,42 @@ prometheus-operator:
...
+        ## redis_used_cpu_sys
+        ## redis_used_cpu_user
+        ## redis_used_memory
+        ## redis_used_memory_overhead
+        ## redis_used_memory_rss
+        ## redis_used_memory_startup
+        - url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.applications.redis
+          writeRelabelConfigs:
+            - action: keep
+              regex: (?:redis_((blocked_|)clients|cluster_enabled|cmdstat_calls|connected_slaves|(evicted|expired|tracking_total)_keys|instantaneous_ops_per_sec|keyspace_(hitrate|hits|misses)|(master|slave)_repl_offset|maxmemory|mem_fragmentation_(bytes|ratio)|rdb_changes_since_last_save|rejected_connections|total_commands_processed|total_net_(input|output)_bytes|uptime|used_(cpu_(sys|user)|memory(_overhead|_rss|_startup|))))
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:1469]: Just a question: Why is the `(blocked_)` at the beginning in parenthesis? Couldn't it be made just `blocked_`?

Comment (justrelax19) [reply deploy/helm/sumologic/values.yaml:1469]: +1

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:1469]: it is `blocked_` or ``, because we need to match `redis_blocked_clients` and `redis_clients`

Comment (pmalek-sumo) [reply deploy/helm/sumologic/values.yaml:1469]: ahh sure, cool

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**